docs/maya/glue.md

```diff
@@ -0,0 +1,145 @@
+| **Glue Resistance**      | 1.0 | Force to preserve the distance to the closest point on the closest neighbor surface. A higher value represents higher correction.<ul><li>*Tip*: Paint a value of 0.0 in those areas where the gluing effect is not needed and it will increase the performance.</li></ul> |
+| **Masses**                      | 1.0 | Multiplier to the individual mass values per vertex. |
+| **Max Glue Distance Multiplier** | 1.0 | Multiplier to the individual values of the max glue distance per vertex. |
+| **Shape Preservation**          | 0.0 | Amount of correction to apply to a vertex to maintain the initial state of the shape formed with the surrounding vertices. |
+| **Stretching Resistance**       | 1.0 | Force to correct the edge lengths if the current length is greater than the rest length. A higher value represents higher correction.<ul><li>*Tip*: To optimize the painting of the weight, flood it to 1.0 as a starting point and tweak some areas later on.</li><li>*Tip*: Smooth the borders by using the Smooth and Flood combination to make sure that there are no discontinuities in the weights map. This will help the simulation to not produce sharp differences in the dynamics of every vertex compared to its connected vertices.</li></ul> |
+
+<!--
+PENDING FIGURE WITH PAINTED WEIGHTS
+-->
+
+## Debugger
+
+In order to better visualize node constraints and attributes in the Maya viewport there is the option to enable the debugger, found in the dropdown menu labeled *Debug* in the Attribute Editor.
+
+To enable the debugger the *Debug* checkbox must be marked. To select the specific feature you would like to visualize, choose it from the list provided in *Features*. The features that can be visualized with the debugger in the AdnGlue node are:
+
+ - **Distance Constraints**: For each pair of vertices forming a constraint a line will be drawn. If the *Triangulate Mesh* option is disabled the debugged lines will align with the edges of the mesh polygons. If the *Triangulate Mesh* option is enabled the debugged lines will align with the edges of the underlying triangulation of the mesh.
+ - **Shape Preservation**: For each vertex with a shape preservation weight greater than 0.0, a line will be drawn from each adjacent vertex to the opposite adjacent vertex.
+ - **Glue Constraint**: A line will be drawn for every vertex to the closest point on the closest neighbor surface.
+
+<!--
+PENDING FIGURE WITH DEBUG FEATURES
+-->
+
+## Advanced
+
+### Inputs
+
+Once the AdnGlue node is created, it is possible to add new inputs and remove currently connected ones.
+
+- **Add inputs**:
+    1. Select one or more mesh nodes to be assigned as inputs to the AdnGlue.
+    2. Select the AdnGlue output mesh.
+    3. Press *Add Inputs* in the AdonisFX menu from the Edit Glue submenu.
+- **Remove inputs**:
```

Comment:
Should we add a NOTE about the paintable maps not being undoable or redoable when adding or remocing inputs?

Reply:
I have added a note